frontend/crd/src/app/util/http-helper.ts

```diff
@@ -14,6 +14,7 @@ export enum Endpoints {
     USERS = 'users',
     CURRENT_USER = 'current-user',
     SUBMISSION = 'tasks/submission',
```

Comment:
change this to submit?

Reply:
i changed the other endpoint to ALL_SUBMISSIONS since the SUBMISSION endpoint matches with submitting an artifact with ArtifactType.SUBMISSION